modules/migration-downloading-logs.adoc

```diff
@@ -10,14 +10,23 @@ You can download the Velero, Restic, and Migration controller logs in the CAM we
 
 .Procedure
 
-. Click the *Options* menu {kebab} of a migration plan and select *Logs*.
-. To download a specific log, select the following:
-
-* *Cluster*: Source or target cluster
-* *Log source*: Velero, Restic, or Migration controller
-* *Pod source*: For example, `velero-_7659c69dd7-ctb5x_`
-
-. Click *Download all logs* to download the Migration controller log and the Velero and Restic logs of the source and target clusters.
+. Log in to the CAM console.
+. Click *Plans* to view the list of migration plans.
+. Click the *Options* menu {kebab} of a specific migration plan and select *Logs*.
```

Comment:
This {kebab} looks to be rendering.

It might be because your assemblies aren't including the common attributes file where this is defined?

```
include::modules/common-attributes.adoc[]
```

Reply:
Very likely. I re-added it.